worker/src/main/java/gov/cms/ab2d/worker/processor/contract/ContractSliceCreatorImpl.java

```diff
@@ -0,0 +1,90 @@
+package gov.cms.ab2d.worker.processor.contract;
```

Comment:
This class would be a good example of the "added complexity" I mentioned above. All the math logic here is pretty hard to follow.